themes/doks/layouts/partials/header/header.html

```diff
@@ -4,19 +4,23 @@
   <div class="banner-message" data-banner-messages-container>
     <!-- Each message must have this data attribute -->
     <div class="banner-message__item" data-banner-message>
-        <b><a style="color:#FFFFFF; text-decoration: underline;" href="https://docs.r3.com/en/platform/corda/5.0-beta.html">Corda 5 Beta</a></b> - Read more about the next-gen Corda.
+      <b><a style="color:#FFFFFF; text-decoration: underline;"
+          href="https://docs.r3.com/en/platform/corda/5.0-beta.html">Corda 5 Beta</a></b> - Read more about the next-gen
+      Corda.
     </div>
     <!-- Uncomment, edit and duplicate as necassary -->
```

Comment:
```suggestion
    <!-- Uncomment, edit and duplicate as necessary.  -->
```
Can we add a maximum number of characters allowed in banner messages?

Reply:
ok added some comment about word count.